.github/workflows/pr-project-board.yml

```diff
@@ -19,4 +20,4 @@ jobs:
 
       - name: Move PR to Project Board for triage
         run: |
-          gh pr edit ${{github.event.pull_request.number}} --add-project "NR1 Quickstarts Community"
+          PR_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH") HEADER="Accept: application/vnd.github.inertia-preview+json" PR_ID=$(gh api -H "$HEADER" "repos/$REPOSITORY/pulls?state=open" --jq ".[] | select(.number == $PR_NUMBER).id") PROJECT=$(gh api -H "$HEADER" "repos/$REPOSITORY/projects" --jq ".[] | select(.id == $PROJECT_ID).id") COLUMN=$(gh api -H "$HEADER" "projects/$PROJECT/columns" --jq ".[] | select(.name == \"$COLUMN_NAME\").id") gh api -H "$HEADER" -X POST "projects/columns/$COLUMN/cards" -f content_type='PullRequest' -F content_id="$PR_ID"
```

Comment:
Would it make more sense to use board automation for this, might be a lot cleaner and easier to manage:
![Screenshot 10-12-2021 13-51-39](https://user-images.githubusercontent.com/17122543/145584428-c8eeb4c9-30c4-4e90-a3df-7037277f7375.png)